### PR TITLE
Adds Shane Rosenthal contributor and Panel Roles plugin

### PR DESCRIPTION
**Pull request by shanerbaner82**

### CODE REVIEW

File: content/plugins/shane-rosenthal-panel-roles.md
Comment (saade) [inline content/plugins/shane-rosenthal-panel-roles.md:1]: Please do not include the word "Filament" in the name or slug of your plugin, or the file names. It is redundant, as all plugins on the website are for Filament.

**Review by saade — CHANGES_REQUESTED** [review]

File: content/plugins/shane-rosenthal-panel-roles.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+name: Panel Roles
+slug: filament-panel-roles
```
Comment (saade) [inline content/plugins/shane-rosenthal-panel-roles.md:3]:
```suggestion
slug: shanerbaner82-panel-roles
```

**Review by saade — CHANGES_REQUESTED** [review]

File: content/plugins/shane-rosenthal-panel-roles.md
Comment (shanerbaner82) [reply content/plugins/shane-rosenthal-panel-roles.md:1]: I think your suggestion fixed it, right?

Comment (saade) [reply content/plugins/shane-rosenthal-panel-roles.md:1]: Yes it does, just explaining why i've made the suggestion

File: content/authors/shane-rosenthal.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: Shane D Rosenthal
+slug: shane-rosenthal
+avatar: laravelonline.jpeg
```
[the remark this answers is not in the document]
Comment (shanerbaner82) [reply content/authors/shane-rosenthal.md]: Updated, good catch.

**Review by awcodes — APPROVED** [review]

**Comment by danharrin** [comment]
Your plugin's `README` is loaded into the plugin page on the Filament website. There is now a duplicate image on your page, as it also exists in your documentation. You can remove it from the Filament website by giving it a `filament-hidden` class, so then it is only shown once.

**Pull request merged**